ext/php_cassandra.c

```diff
@@ -8,6 +9,8 @@
 #include "types/map.h"
 #include "types/set.h"
 
+#define PHP_CASSANDRA_DEFAULT_LOG "php-driver.log"
```

Comment:
It's in the current working directory (could be the VCWD too). I think for something like "error_log()" the SAPI determines the directory where the PHP logs go. I don't think we have access to that information here.